spec/features/bulk_resource_delete_spec.rb

```diff
@@ -0,0 +1,126 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Bulk delete can be run on search results' do # rubocop:disable RSpec/DescribeClass
+  let(:exhibit) { create(:exhibit, published: true) }
+  let(:curator) { create(:exhibit_curator, exhibit: exhibit) }
+  let(:resource_1) { DlmeJson.new(json: json_1, metadata: metadata_1, exhibit: exhibit) }
+  let(:resource_2) { DlmeJson.new(json: json_2, metadata: metadata_2, exhibit: exhibit) }
+  let(:resource_3) { DlmeJson.new(json: json_3, metadata: metadata_3, exhibit: exhibit) }
+  let(:resource_4) { DlmeJson.new(json: json_4, metadata: metadata_4, exhibit: exhibit) }
+  let(:fixture_file_path_1) { File.join(fixture_path, 'json/bodleian.json') }
+  let(:fixture_file_path_2) { File.join(fixture_path, 'json/embeddable.json') }
+  let(:fixture_file_path_3) { File.join(fixture_path, 'json/iiif-single-image.json') }
+  let(:fixture_file_path_4) { File.join(fixture_path, 'json/long-description.json') }
+  let(:json_1) { File.open(fixture_file_path_1).read }
+  let(:json_2) { File.open(fixture_file_path_2).read }
+  let(:json_3) { File.open(fixture_file_path_3).read }
+  let(:json_4) { File.open(fixture_file_path_4).read }
+  let(:metadata_1) do
+    { 'traject_context_command_line.filename' => fixture_file_path_1,
+      'traject_context_source' => 'matching_resource' }
+  end
+  let(:metadata_2) do
+    { 'traject_context_command_line.filename' => fixture_file_path_2,
+      'traject_context_source' => 'not_in_the_original_search_results' }
+  end
+  let(:metadata_3) do
+    { 'traject_context_command_line.filename' => fixture_file_path_3,
+      'traject_context_source' => 'not_in_the_original_search_results' }
+  end
+  let(:metadata_4) do
+    { 'traject_context_command_line.filename' => fixture_file_path_4,
+      'traject_context_source' => 'matching_resource' }
+  end
```

Comment:
sorry, this is rather verbose and repetitive.  happy to revisit and find some way to make more concise if people want.  but i did want the tests to show that exactly the subset of things in the search results gets deleted.